index.js

```diff
@@ -1,13 +1,27 @@
 function isPalindrome(word) {
-  // Write your algorithm here
+  const reversedArray = []
+  const wordArray = word.split('')
+  for(let i = wordArray.length -1; i >= 0; i--){
+    reversedArray.push(wordArray[i])
+  }
+  const reversedWord = reversedArray.reduce((acc, item) => acc+item)
+  return word === reversedWord ? true : false 
```

Comment:
What primitive data type does `===` return?

Reply:
boolean

Reply:
Right. So what can you do to refactor this?

Reply:
Gotcha. Just write <<return word === reversedWord>> and skip the <<? true : false>>

Reply:
Correct. Did you see my PR comment above? It would be great if you could take a few minutes to refactor.

Reply:
On it